src/test/spec/index.ts

```diff
@@ -1,6 +1,6 @@
-import * as glob from 'glob';
-import * as Mocha from 'mocha';
-import * as path from 'path';
+import glob from 'glob';
+import Mocha from 'mocha';
+import path from 'path';
```

Comment:
I was wondering what is the difference between this import and the namespace imports? Is it the same?

Reply:
Wow this is a good question I am looking into it right now reverting to the namespace import for all 3 imports I am getting an error message only `import * as Mocha from 'mocha'` for and for `import * as glob from 'glob'` I might have corrected the 3 imports without thinking but I will look into the difference between `import * as path from 'path'` and  `import path from 'path'` right now I didn't think further after all the test passed...


***`import * as Mocha from 'mocha'` ***:

```log 
This expression is not constructable.
  Type 'typeof Mocha' has no construct signatures.ts(2351)
index.ts(2, 1): Type originates at this import. A namespace-style import cannot be called or constructed,
and will cause a failure at runtime. Consider using a default import or import require here instead.
```

***`import * as glob from 'glob'`***:

```log
This expression is not callable.
  Type 'typeof G' has no call signatures.ts(2349)
index.ts(1, 1): Type originates at this import. A namespace-style import cannot be called or constructed,
and will cause a failure at runtime. Consider using a default import or import require here instead.
```

Reply:
With `import * as path from 'path'`:
(alias) namespace path
(alias) const path: PlatformPath
import path

With `import path from 'path'`:
(alias) namespace path
(alias) const path: PlatformPath
import path

using `    "@types/node": "14.6.4"`:
![Screenshot_20201104_202248](https://user-images.githubusercontent.com/42672814/98185904-9916a000-1edb-11eb-9cad-eb95a9819fe5.png)

Reply:
So the reason why it's not breaking is because the `--allowSyntheticDefaultImports` is set to true

with `"allowSyntheticDefaultImports": false` it would not allow the fix for 'glob' and 'mocha' and only `import * as path from 'path'` would be valid and `import path from 'path'` would yield this error:

```log
import path
Module '"path"' can only be default-imported using the 'esModuleInterop' flag ts(1259)
path.d.ts(152, 5): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

Reply:
the default for `--allowSyntheticDefaultImports` is:

```
module === "system" || --esModuleInterop
```

I didn't know I always set it to `true` but in this case, it is `true` by default as per [TS HandBook ― Compiler Options](https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options)

Reply:
Thanks for clarifying! It probably makes imports of CommonJS modules easier, I like it 👍